app/controllers/grant_submissions/submissions_controller.rb

```diff
@@ -39,10 +39,18 @@ def edit
     def create
       submission
```

Comment:
what this submission method does? If its setting submission,  I would rename as set_submission instead of submission, for better readability.